lib/DebugInfo/Symbolize/SymbolizableObjectFile.cpp

```diff
@@ -255,9 +255,32 @@ DIInliningInfo SymbolizableObjectFile::symbolizeInlinedCode(
     uint64_t ModuleOffset, FunctionNameKind FNKind, bool UseSymbolTable) const {
   DIInliningInfo InlinedContext;
 
-  if (DebugInfoContext)
+  if (DebugInfoContext){
     InlinedContext = DebugInfoContext->getInliningInfoForAddress(
         ModuleOffset, getDILineInfoSpecifier(FNKind));
+
+    // HACK: This strips projectRoot in the bugsnag-expected manner (on all frames).
+    // HACK: Upstream doesn't have the getCompilationDirectory() function.
+    if (InlinedContext.getNumberOfFrames() != 0) {
```

Comment:
apply [this hack](https://github.com/bugsnag/llvm/commit/2863b3b3ee5dd15a202f9dc55230e7b724d07ffe) to every returned frame